assets/course-theme/focus-mode.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Focus mode class name and session storage key.
+ *
+ * @type {string}
+ */
+const FOCUS_MODE_CLASS = 'sensei-course-theme--focus-mode';
+
+/**
+ * Initialize focus mode state on page load.
+ */
+const initFocusMode = () => {
+	restoreFocusModeState();
+	setTimeout( () => {
+		document.body.classList.add( `${ FOCUS_MODE_CLASS }--animated` );
+	}, 500 );
```

Comment:
This is in order to show the animation also when loading for first time, right? Is this something really needed? I haven't seen it, but on first thought it seems weird to me.

Reply:
It's to **not** animate when restoring the collapsed state during page load (so it's switched to immediately, like it's the initial state), and then enable animations when the user is clicking the toggle.

Reply:
Okay, that makes much more sense!
Even thou I do not understand how it works maybe you can explain it to me some day :P